modules/test/playwright/tests/users-admin-web/defaultUserAssociations.spec.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {instanceSettingsPagesTest} from '../../fixtures/instanceSettingsPagesTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {usersAndOrganizationsPagesTest} from '../../fixtures/usersAndOrganizationsPagesTest';
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	loginTest(),
+	instanceSettingsPagesTest,
+	usersAndOrganizationsPagesTest
+);
+
+test('LPD-30006 edit configuration users default user associations CP', async ({
+	apiHelpers,
+	defaultUserAssociationsPage,
+	editUserPage,
+	instanceSettingsPage,
+	page,
+	usersAndOrganizationsPage,
+}) => {
+	const userGroup = await apiHelpers.headlessAdminUser.postUserGroup();
```

Comment:
we are not deleting this after the test.
I think the best solution is to add the ```if (this.apiHelpers instanceof DataApiHelpers) {...```  block in the `postUserGroup` method (have a look to the previous method on how it's implemented) and the deletion of it (inside the `apiHelper`)